vendor/modules.txt

```diff
@@ -1115,6 +1115,9 @@ github.com/waku-org/go-zerokit-rln-x86_64/libs/x86_64-pc-windows-gnu
 github.com/waku-org/go-zerokit-rln-x86_64/libs/x86_64-unknown-linux-gnu
 github.com/waku-org/go-zerokit-rln-x86_64/libs/x86_64-unknown-linux-musl
 github.com/waku-org/go-zerokit-rln-x86_64/rln
+# github.com/waku-org/sds-go-bindings v0.0.0-20250724095249-05e797c76b8c
+## explicit; go 1.22.10
```

Comment:
I guess this part also would need adjustment since we no longer build with go 1.22 on CI.
We upgraded go to 1.23 a while ago.